src/openepd/api/base_sync_client.py

```diff
@@ -337,15 +340,15 @@ def do_request(
         self,
         method: str,
         endpoint: str,
-        params=None,
-        data=None,
-        json=None,
-        files=None,
-        headers=None,
+        params: dict[str, Any] | None = None,
+        data: dict[str, Any] | str | bytes | None = None,
+        json: dict[str, Any] | None = None,
+        files: dict[str, Any] | None = None,
```

Comment:
Is this definition correct? I don't have any objection, just looks suspicious. What's in values?